package.json

```diff
@@ -30,6 +30,8 @@
     "npm": ">=7.0.0",
     "node": ">=16 <17"
   },
-  "dependencies": {},
+  "dependencies": {
+    "react-icons": "^4.7.1"
```

Comment:
You should remove this from here and install it only inside "app" 🙏🏼

Reply:
Done! :raised_hands: